crates/polars-plan/src/dsl/expr.rs

```diff
@@ -71,6 +71,7 @@ pub enum Expr {
     Column(ColumnName),
     Columns(Arc<[ColumnName]>),
     DtypeColumn(Vec<DataType>),
+    IndexColumn(Vec<i32>),
```

Comment:
Can we make this an `Arc<[i32]>`? That makes the clone cheap.

Reply:
Done :)